api/src/main/java/io/strimzi/api/kafka/model/rebalance/BrokerAndVolumeIds.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.rebalance;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.api.kafka.model.common.UnknownPropertyPreserving;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Configures the brokerId and VolumeId for the remove-disks endpoint for Cruise Control
+ */
+@Buildable(
+        editableEnabled = false,
+        builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonPropertyOrder({"brokerId", "volumeIds"})
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+@ToString
+public class BrokerAndVolumeIds implements UnknownPropertyPreserving {
+
+    private Integer brokerId;
+    private List<Integer> volumeIds;
+    private Map<String, Object> additionalProperties;
+
+    @Description("Id of the broker which contains the disk from which you want to move the the partition replicas from")
```

Comment:
Here and in the other doc comments -> I think we use ID in the docs. `Id` is only the Java capitalization.

Reply:
Okay, I will correct it